q2_feature_table/_summarize/summarize_assets/overview.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+    <link rel="stylesheet" href="./q2templateassets/css/bootstrap.min.css">
+    <link rel="stylesheet" href="./q2templateassets/css/normalize.css">
+    <script src="./app/js/child.js" charset="utf-8"></script>
+  </head>
+  <body>
+    <div class="row">
```

Comment:
That's it, the `q2templates` has the container, and this was the weird hacked together combination of it to be a child, which forgot the container div. Should that `q2templates` child frame we had talked about happen before merging this?